docuilib/src/css/custom.css

```diff
@@ -29,3 +29,16 @@
 html[data-theme='dark'] .docusaurus-highlight-code-line {
   background-color: rgba(0, 0, 0, 0.3);
 }
+
+.main-tabs {
```

Comment:
maybe move this style to a scss files of the ComponentPage

Reply:
It's not working...